### DISCUSSION TITLE
Other themes (like Arya-Orange)

### DISCUSSION THREAD

**Discussion by ghost** [primefaces/community#1382] [Figma UI Kit] [answered]
Hello.
We are planning to buy the Figma UI Kit but we want to solve the following doubt before doing it,

It  says that it only ships with two themes: "Lara Blue Light" and "Lara Blue Dark", can I switch the theme in Figma somehow to have the "Arya Orange" (or others)?

Thanks in advance.

**Comment by ren-x7** [comment]
@onursenture I wanna know too, would it be that hard to match with Arya themes? Thanks.

**Comment by onursenture** [comment]
Hi, The Arya theme is out of the scope of the Figma UI Kit. Apart from Lara, Aura is the only theme that will be added. Therefore, it's unfortunate, but the inclusion of Arya in the Figma UI Kit is not under consideration.

**Reply by ghost** [reply]
Okay, it is clear that the Arya theme is not going to be included in any way, but is there any workaround / possibility that we can add it ourselves?

**Reply by onursenture** [reply, marked as answer]
Yes, if you're using the latest version of UI Kit, it supports tokens through Tokens Studio. You can switch to the Arya theme by changing all token values using Arya's _variables.scss (for example, for PrimeNG, this file: https://github.com/primefaces/primeng-sass-theme/blob/0affe78dc1b945eaa6bac733042ba6a76fd55b0d/themes/arya/_variables.scss#L4) as a reference. After doing this, you should review all components and manually fix any remaining gaps.

**Reply by ghost** [reply]
Thanks for the explanation but, unfortunately, I am not able to see that file:

![image](https://github.com/primefaces/community/assets/71069615/bc5cff3b-4837-4344-a629-8141647fefe2)

**Reply by onursenture** [reply]
Fixed.
